pytest_voluptuous/plugin.py

```diff
@@ -9,13 +9,40 @@ def pytest_assertrepr_compare(op, left, right):
     if isinstance(left, S) and (op == '<=' or op == '==') or isinstance(right, S) and op == '==':
         if isinstance(left, S):
             source = left
+            data = right
         else:
             source = right
+            data = left
+
         if isinstance(source.error, MultipleInvalid):
-            errors = ['- {}: {}'.format('.'.join(map(str, error.path)), error) for error in source.error.errors]
+            errors = [format_error(error, data) for error in source.error.errors]
         else:
-            errors = ['- {}: {}'.format('.'.join(map(str, source.error.path)), source.error)]
+            errors = [format_error(source.error, data)]
+
         return [
             'failed to validation error(s):'
         ] + errors
     return None
+
+
+def format_error(error, data):
+    if error.path:
+        prefix = '.'.join(map(str, error.path)) + ': '
+        value = get_value(data, error.path)
+        suffix = (' (value: ' + repr(value) + ')') if value else ''
```

Comment:
This will skip values that are explicitly set to `None`. I think we need to return a sentinel object from `get_value` and only skip if sentinel gets returned (ie. path is not found from `data`)?

Reply:
Also, maybe rename `value:` as `actual:` - I think it's more common convention.

Reply:
done